bin/needs-a-genny

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+
+EXERCISES = File.expand_path('../../exercises', __FILE__)
+
+genny_path = ->(slug) {
+  File.join(EXERCISES, slug, '.meta', 'generator')
+}
+
+genny_path_exists = ->(slug) { Dir.exist?(genny_path[slug]) }
+
+puts Dir.children(EXERCISES)
+  .reject(&genny_path_exists).sort
```

Comment:
```suggestion
        .reject(&genny_path_exists).sort
```

Reply:
The argument given to `puts` might be better named as to what it represents.
`puts doohicky` where "doohicky" is a good name for whatever that argument to puts is doing.

Reply:
gave it a name